src/instructlab/model/chat.py

```diff
@@ -635,11 +646,11 @@ def start_prompt(
             self.multiline_mode = 0
             self.multiline = not self.multiline
 
-        # Optional parameters
-        create_params = {}
-        if self.greedy_mode:
+        # Temperature parameters
+        create_params = {
```

Comment:
Nit: Initialize an empty dict for `create_params` first, and then add temperature for code readability purposes.